sites/example-project/src/pages/+layout.js

```diff
@@ -131,19 +131,6 @@ export const load = async ({ fetch, route, params, url }) => {
 			})();
 		}
 
-		// Pre-render columns
-		try {
-			// Since we're running in node here (not the browser), this function is synchronous and doesn't need to be awaited
-			usqlQuery(`DESCRIBE ${sql}`, {
-				route_hash: routeHash,
-				additional_hash: paramsHash,
-				query_name: `${query_name}__DESCRIBE`,
-				prerendering: building
-			});
-		} catch (e) {
-			console.debug('Failed to pre-render columns', e instanceof Error ? e.message : e);
-		}
-
```

Comment:
Doesn't this disable column pre-rendering completely?

Reply:
No. Adding the `__DESCRIBE` query was actually unnecessary (as far as I can tell). Using the existing `_columns` query from `Query` is sufficient. Otherwise, we end up running a nested `DESCRIBE DESCRIBE SELECT ...` when we try to pre-render the columns for the columns query.

All that was necessary for column pre-rendering is using the proper query in `process-queries`
https://github.com/evidence-dev/evidence/blob/7ae11a804e4b93b9325b157c1d31085223928eb0/packages/lib/preprocess/src/process-queries.cjs#L74